src/gui/transferlistsortmodel.h

```diff
@@ -64,7 +66,9 @@ class TransferListSortModel final : public QSortFilterProxyModel
 
     TorrentFilter m_filter;
     mutable CachedSettingValue<int> m_subSortColumn;
+    mutable CachedSettingValue<int> m_subSortOrder;
     mutable int m_lastSortColumn = -1;
+    mutable int m_lastSortOrder = 0;
```

Comment:
`TransferListSortModel::sort` is not const, it should work without marking these variables `mutable`.

Reply:
removed (×4)